src/Symfony/Component/Notifier/Bridge/Smsmode/SmsmodeTransport.php

```diff
@@ -69,7 +69,11 @@ protected function doSend(MessageInterface $message): SentMessage
         }
 
         $response = $this->client->request('POST', $endpoint, [
-            'headers' => ['X-Api-Key' => $this->apiKey],
+            'headers' => [
+                'X-Api-Key' => $this->apiKey,
+                'Content-type' => 'application/json',
```

Comment:
tis content-type is added automatically when using the `json` option to build the body, so it is useless to add it explicitly.